app/models/post_mover.rb

```diff
@@ -701,7 +701,9 @@ def close_topic_and_schedule_deletion
     @original_topic.update_status("closed", true, @user)
 
     days_to_deleting = SiteSetting.delete_merged_stub_topics_after_days
-    if days_to_deleting > 0
+    if days_to_deleting == -1
+      @original_topic.trash!
+    elsif days_to_deleting > 0
```

Comment:
This seems backwards... I would've expected `0` for 'delete immediately' and `-1` for 'disable the feature' 🤔

Reply:
Ah I just saw your comment :D yes I have the same question

Reply:
Yeah that sounds a lot more logical! I didn't want to mess with existing setting of 0 = never but I will leave this to the engineering team.

> This seems backwards... I would've expected `0` for 'delete immediately' and `-1` for 'disable the feature' 🤔

Yeah that sounds a lot more logical! I didn't want to mess with existing setting of 0 = never but I will leave this to the engineering team.

Reply:
It will need a migration, since 0 has a different meaning at the moment. But I think it's worth it.

(same discussion happening in `t/134344/8`)